compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1499,35 +1499,26 @@ OMR::ARM64::TreeEvaluator::vfmaEvaluator(TR::Node *node, TR::CodeGenerator *cg)
 
 enum VectorCompareOps : int32_t
    {
-   EQ,
-   NE,
-   LT,
-   LE,
-   GT,
-   GE,
+   VECTOR_COMPARE_INVALID,
```

Comment:
`VECTOR_COMPARE_INVALID` is not used anywhere in this PR.  What is the purpose of this symbol?

Reply:
It makes the indexing of `vectorCompareOpCodes` consistent. Since TR::DataTypes enum has `NoType` as 0, `VECTOR_COMPARE_INVALID` is needed to describe the index of the first and second dimension of the array in the same way as shown below.
```
      op = vectorCompareZeroOpCodes[compareOp - 1][elemType - 1];
```
https://github.com/eclipse/omr/blob/647678f93a84d1b2789c8432d8991adcac7b3773/compiler/il/OMRDataTypes.hpp#L237-L266